pcdsdevices/pseudopos.py

```diff
@@ -1013,10 +1013,18 @@ def forward(self, pseudo_pos: tuple) -> tuple:
         pseudo_field, = self.PseudoPosition._fields
         real_field, = self.RealPosition._fields
 
+        xp = self._table_data_by_name[pseudo_field]
+        fp = self._table_data_by_name[real_field]
+
+        # xp must be increasing
+        if not xp[1] > xp[0]:
```

Comment:
This is very intolerant of noise in the signal, are we confident we won't jitter enough to trigger this by accident?

Reply:
These come from fixed values in a text file that can look something like this:
```
50        71
55        41.5
60        18.8
65        6
70        1.3
75        0.5
```
Since they are fixed values there won't be any jitter, but I think it's worth adding a more thorough check so we can throw a warning if the values aren't monotonic